shared/reducers/unlock-folders.js

```diff
@@ -8,29 +8,54 @@ import type {UnlockFolderActions, Device} from '../constants/unlock-folders'
 export type State = {
   phase: 'dead' | 'promptOtherDevice' | 'paperKeyInput' | 'success',
   devices: ?Array<Device>,
-  paperkey: ?HiddenString,
   paperkeyError: ?HiddenString
 }
 
 const initialState: State = {
   phase: 'dead',
   devices: null,
-  paperkey: null,
   paperkeyError: null
 }
 
 export default function (state: State = initialState, action: UnlockFolderActions): State {
   // TODO: Fill out the rest of this reducer
   switch (action.type) {
-    case Constants.toPaperKeyInput:
+    case Constants.loadDevices:
       if (action.error) {
         return state
       } else {
+        const devices = action.payload.devices.map(({name, type, deviceID}) => ({
+          type: type === 'mobile' ? 'mobile' : 'desktop',
```

Comment:
if you add the type can you add it to https://github.com/keybase/client/blob/master/shared/login/register/select-other-device/index.render.js.flow#L6
also my code references paper key but this only have mobile/desktop...